Run failed: Publish to WinGet

### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
not relevant
```


### Description

been getting these failed build emails for the last three days in a row

### Logged Error Messages

```text
see screenshot
```


### Screenshots

![image](https://user-images.githubusercontent.com/6933240/215403800-36806e23-4c67-47a7-8a67-38defccce7eb.png)
@vedantmgoyal2009  @ilike2burnthing  this falls into your area or expertise?

failing again, file not found?

https://github.com/Jackett/Jackett/actions/runs/5805503211/job/15756809655
```
Run vedantmgoyal2009/winget-releaser@v2
Updating manifests and creating pull request...
  Executing command: java -jar komac.jar update --id 'Jackett.Jackett' --version 0.21.602 --urls 'https://github.com/Jackett/Jackett/releases/download/v0.21.602/Jackett.Installer.Windows.exe' --submit
  CI environment detected! Komac will throw errors instead of prompting on invalid input
  Latest version of Jackett.Jackett: 0.21.589
  # Created with WinGet Releaser v2 using Komac v1.9.1
  # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
  
  PackageIdentifier: Jackett.Jackett
  PackageVersion: 0.21.602
  InstallerLocale: en-US
  InstallerType: inno
  Scope: machine
  InstallModes:
  - interactive
  - silent
  - silentWithProgress
  UpgradeBehavior: install
  ReleaseDate: 2023-08-09
  Installers:
  - Architecture: x86
    InstallerUrl: https://github.com/Jackett/Jackett/releases/download/v0.21.602/Jackett.Installer.Windows.exe
    InstallerSha256: 3E480200A33A1B5D97233994AECC6D9EC1A4FE6DD4A713C960A8D7D67607A46A
  ManifestType: installer
  ManifestVersion: 1.4.0
  
  # Created with WinGet Releaser v2 using Komac v1.9.1
  # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
  
  PackageIdentifier: Jackett.Jackett
  PackageVersion: 0.21.602
  PackageLocale: en-US
  Publisher: Jackett
  PublisherUrl: https://github.com/Jackett/Jackett
  PublisherSupportUrl: https://github.com/Jackett/Jackett/issues
  Author: Jackett Contributors
  PackageName: Jackett
  PackageUrl: https://github.com/Jackett/Jackett
  License: GPL-2.0
  LicenseUrl: https://github.com/Jackett/Jackett/blob/master/LICENSE
  Copyright: Copyright (C) 1989, 1991 Free Software Foundation, Inc.
  	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280)
  	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
  	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
  	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
  	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
  	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
  	at commands.QuickUpdate.run(QuickUpdate.kt:[11](https://github.com/Jackett/Jackett/actions/runs/5805503211/job/15756809655#step:2:12)3)
  	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:306)
  	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:319)
  	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:40)
  	at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:489)
  	at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:486)
  	at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:506)
  	at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:513)
  	at MainKt.main(Main.kt:24)
  Caused by: java.io.FileNotFoundException: https://api.github.com/repos/Jackett/winget-pkgs/branches/refs/heads/Jackett.Jackett-0.21.602-2374854D1C892FDB0A97C229A5376EF13BE01403
  	at org.kohsuke.github.GitHubConnectorResponseErrorHandler$1.onError(GitHubConnectorResponseErrorHandler.java:60)
  	at org.kohsuke.github.GitHubClient.detectKnownErrors(GitHubClient.java:473)
  	at org.kohsuke.github.GitHubClient.sendRequest(GitHubClient.java:435)
  	... 23 more
  node:internal/errors:841
    const err = new Error(message);
                ^
  
  Error: Command failed: & C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\17.0.8-7\x64\bin\java -jar komac.jar update --id 'Jackett.Jackett' --version 0.21.602 --urls 'https://github.com/Jackett/Jackett/releases/download/v0.21.602/Jackett.Installer.Windows.exe' --submit
      at checkExecSyncError (node:child_process:828:11)
      at execSync (node:child_process:899:[15](https://github.com/Jackett/Jackett/actions/runs/5805503211/job/15756809655#step:2:16))
      at D:\a\_actions\vedantmgoyal[20](https://github.com/Jackett/Jackett/actions/runs/5805503211/job/15756809655#step:2:21)09\winget-releaser\v2\dist\index.js:79:39
      at processTicksAndRejections (node:internal/process/task_queues:96:5) {
    status: 1,
    signal: null,
    output: [ null, null, null ],
    pid: [38](https://github.com/Jackett/Jackett/actions/runs/5805503211/job/15756809655#step:2:39)80,
    stdout: null,
    stderr: null
  }
```

--- a/.github/workflows/winget.yml
+++ b/.github/workflows/winget.yml
@@ -8,6 +8,8 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
+      - run: Start-Sleep -Seconds 300 # 5 minutes
+        shell: pwsh
       - uses: vedantmgoyal2009/winget-releaser@v2
         with:
           identifier: Jackett.Jackett

